### PR TITLE
Add payment token tasks for account scope

### PR DESCRIPTION
**Pull request by havan**
This PR adds tasks to payment token related transactions: `add`, `remove`, and `list`.

### CODE REVIEW

File: tasks/account.js [hunk 1]
```diff
@@ -228,6 +228,62 @@ ACCOUNT_SCOPE.task("bot:remove", "Remove bot from the CMAccount")
         }
     });
 
+ACCOUNT_SCOPE.task("payment-token:add", "Add payment token to CMAccount")
+    .addOptionalParam("privateKey", "Private key to use", process.env.CMACCOUNT_PK)
```
Comment (mo-c4t) [inline tasks/account.js:232]: Why this is optional param? The privatekey is required to perform the transaction, isn't it?

File: tasks/account.js [hunk 2]
```diff
@@ -228,6 +228,62 @@ ACCOUNT_SCOPE.task("bot:remove", "Remove bot from the CMAccount")
...
+    .addOptionalParam("cmAccount", "CMAccount address", process.env.CMACCOUNT_ADDRESS)
+    .addParam("paymentToken", "Payment token address")
+    .setAction(async (taskArgs, hre) => {
+        const cmAccount = await getCMAccount(taskArgs.cmAccount);
```
Comment (mo-c4t) [inline tasks/account.js:236]: What happens if cmAccount is not given?
Also what happens if params given pk and address are not for the same account?

Comment (havan) [reply tasks/account.js:236]: It just fails. Same for the private key. 

It's just more convenient when you set `CMACCOUNT_PK` and `CMACCOUNT_ADDRESS` in your envs (ie: `export CMACCOUNT_ADDRESS=...` etc) and then you can just run the command without providing `--cm-account` and `--private-key`.

File: tasks/account.js (hunk 1 above)
Comment (havan) [reply tasks/account.js:232]: So it's possible to set the environment variable `CMACCOUNT_PK` and not provide the `--private-key` param from the command line. When you make it optional, you can provide a default value, and for that I've provided `process.env.CMACCOUNT_PK`.

File: tasks/account.js (hunk 2 above)
Comment (havan) [reply tasks/account.js:236]: > Also what happens if params given pk and address are not for the same account?

Transaction fails

Comment (mo-c4t) [reply tasks/account.js:236]: Shouldn't we add a check and return an error describing the issue? Or connect(signer) returns a proper error?

Comment (havan) [reply tasks/account.js:236]: I don't think it's needed. The user of these functions should know enough to interpret the error messages. 

And btw, when I say transaction fails, it fails off-chain, it's not send to the node and doesn't cost anything. Ethers library simulates the transaction and errors out before sending the transaction to be mined. 

Example output:
```
<----------8<---------->
❌ Transaction failed!
Message: execution reverted
Reason: AccessControlUnauthorizedAccount (0xFe77dcE375C3814F15F8035bCAC1A791D3dCdf21,0x9a95e87c5af084bf5db8491c3a6515da9dd6da39b24b0eb0af08d7b9cd808d91)
```

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**